envisage/resource/tests/resource_manager_test_case.py

```diff
@@ -43,15 +51,15 @@ class ResourceManagerTestCase(unittest.TestCase):
     def setUp(self):
         """ Prepares the test fixture before each test method is called. """
 
-        self.stored_urlopen = urllib2.urlopen
-        urllib2.urlopen = stubout_urlopen
+        self.stored_urlopen = url_library.urlopen
```

Comment:
You're right, this can be left alone for now; it is a separate issue. Probably not worth another immediate PR either.